napari/layers/utils/color_transformations.py

```diff
@@ -74,20 +74,18 @@ def transform_color_cycle(
     -------
     transformed : cycle
         cycle of Nx4 numpy arrays with a dtype of np.float32
+    transformed_color_cycle: np.ndarray
```

Comment:
does the return annotation need to be updated here?

Reply:
Good point. I also think the variable names were confusing so I updated them.